pywinauto/base_wrapper.py

```diff
@@ -8,19 +8,20 @@
 import win32gui
 import win32process
 import locale
+import abc
 
-from .. import SendKeysCtypes as SendKeys
-from .. import six
-from .. import win32defines, win32structures, win32functions
-from ..timings import Timings
-from ..actionlogger import ActionLogger
-from .. import handleprops
-from .. import mouse
-from ..mouse import _perform_click_input
-from .. import backend
+from . import SendKeysCtypes as SendKeys
+from . import six
+from . import win32defines, win32structures, win32functions
+from .timings import Timings
+from .actionlogger import ActionLogger
+from . import handleprops
+from .mouse import _perform_click_input
+#from . import backend
 
 #=========================================================================
 def remove_non_alphanumeric_symbols(s):
```

Comment:
The function handles only space symbols so it should be renamed to ```remove_white_space_symbols``` or changed to support other non-alphanumeric symbols.